### ISSUE TITLE
Support GIF files

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Rustemt/metadata-extractor#64]
```
http://en.wikipedia.org/wiki/Graphics_Interchange_Format

http://www.prepressure.com/library/file-formats/gif

http://www.fileformat.info/format/gif/egff.htm

IMPORTANT NOTE

I need to consider whether it's viable to implement support for GIF files, 
given that the Unisys believes the LZW algorithm to be proprietary. Some say 
that only GIF encoders require license fees be paid, but this still requires 
some analysis. If you have any experience in this field, please comment on this 
issue. Thank you.
```

Original issue reported on code.google.com by `drewnoakes` on 29 Oct 2012 at 3:59

**Comment by GoogleCodeExporter** [comment]
```
Further analysis of the Unisys's claimed patent with regards to GIF indicates 
that it applies to the LZW algorithm used to encode the image data. Extracting 
data from the header of GIF files does not fall under that patent.
```

Original comment by `drewnoakes` on 31 Dec 2012 at 3:41
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This has been implemented and will be included in 2.7.0.

Image dimensions, colour depth, as well as palette size is included.

See commit 897708fdac9e3fd956622ce5b6abbc3c2fd665bc for details.
```

Original comment by `drewnoakes` on 31 Dec 2012 at 5:10
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Testing whether it's possible to link to revisions from comments:

r897708fdac9e3fd956622ce5b6abbc3c2fd665bc
```

Original comment by `drewnoakes` on 31 Dec 2012 at 5:11
